webapp/src/main/resources/sass/mojito.scss

```diff
@@ -479,6 +488,10 @@ div.targetstring-container textarea {
     width: 60%;
 }
 
+.history-initial {
+    color: $brand-primary;
```

Comment:
I guess the primary color should be used for "important" things. Here it seems to be a minor information, should we use: light gray + italic?

Reply:
done